.gitignore

```diff
@@ -86,3 +86,9 @@ _deps
 *.exe
 *.out
 *.app
+
+# Cpp App
+*build-src-cpp-structure*
+src-cpp-structure/projects/App/translations.qrc
```

Comment:
Why you add `qrc` to ignore files?

Reply:
Actually you can ignore that, when I started wanted to add different way for adding translations and that's why I added `translation.cmake` and `check_translations` cmake function there, but later I found out that we have some translation script and `.go` script for building `.rcc` file.